### PR TITLE
OpenFF Sulfur Optimization Training Coverage Supplement v1.0

### PR DESCRIPTION
**Pull request by ntBre**
This is a dataset for increasing coverage on the groups outlined [here](https://openforcefield.atlassian.net/wiki/spaces/FF/pages/2606465079/Increasing+dataset+coverage#To-do). As mentioned in the README, these were selected from ChEMBL 34 based on the SMIRKS patterns in `sulfur.dat`. If this looks good, I have an additional 5 molecules for each label that I will submit as a separate dataset for use in benchmarking with the same input scripts.

## New Submission Checklist

- [x] Created a new folder in the submissions directory containing the dataset
- [x] Added `README.md` describing the dataset see [here](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2020-03-26-OpenFF-Gen-2-Torsion-Set-6-supplemental-2) for examples
- [x] All files used to produce the dataset are included with a description
- [x] Dataset follows the QCSubmit schema defined for [Datasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L340), [OptimizationDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1062) and [TorsionDriveDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1225)
- [x] Dataset filename matches pattern `dataset*.json`; may feature a compression extension, such as `.bz2`
- [x] A PDF depicting the molecules is attached, in the case of torsiondrives this should include the highlighting of the central bond, this can be done automatically using [qcsubmit](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L854). 
- [x] QCSubmit validation passed
- [x] Made a new dataset entry in the mapping table in repository `README.md`
- [ ] Ready to submit!

### CODE REVIEW

File: submissions/2024-09-11-OpenFF-Sulfur-Optimization-Training-Coverage-Supplement-v1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# OpenFF Sulfur Optimization Training Coverage Supplement v1.0
+
+## Description
+
+An optimization data set created to improve the training coverage of sulfonic
+and phosphonic acids, sulfone, sulfonate, sulfinyl, sulfoximine, sulfonamides,
+thioether, and 1,3-thiazole groups. The molecules in this data set were selected
+from the ChEMBL 34 database.
```
Comment (lilyminium) [inline submissions/2024-09-11-OpenFF-Sulfur-Optimization-Training-Coverage-Supplement-v1.0/README.md:8]: Are these full molecules from the CHEMBL database or just fragments? If they're fragments, could you please briefly describe the fragmentation and selection procedure? (e.g. on rotatable bonds, RECAP -- if they were clustered and selected for diversity, then which metric was used, if size was a consideration, etc).

The link below to a commit version of curato is fantastic (it fixes a lot of woes that happen when people point to a branch they later update!), but it's not clear from just the repo itself what was done to select molecules.

**Review by lilyminium — COMMENTED** [review]
Largely LGTM, thanks @ntBre! Just one request to be a bit more explicit how the molecules were chosen.

I think this PR also needs CI to be running (e.g. PR #375 or #377) before it can submit / start the error cycle.

File: submissions/2024-09-11-OpenFF-Sulfur-Optimization-Training-Coverage-Supplement-v1.0/README.md (hunk 1 above)
Comment (ntBre) [reply submissions/2024-09-11-OpenFF-Sulfur-Optimization-Training-Coverage-Supplement-v1.0/README.md:8]: These are actually full molecules even though they're pretty small. I clustered them in a couple of cases but mostly just sorted by size and manually scrolled through the first ~500 or so molecules picking out ones that caught my eye. That sounded pretty arbitrary to me, so I didn't think to add more to the description, but I'm happy to do so now if that sounds reasonable. Maybe something like

The molecules in this data set were manually selected from a subset of the smallest matching structures in the ChEMBL 34 database.

Comment (lilyminium) [reply submissions/2024-09-11-OpenFF-Sulfur-Optimization-Training-Coverage-Supplement-v1.0/README.md:8]: Yup, that's a lot clearer, thanks!

Comment (lilyminium) [reply submissions/2024-09-11-OpenFF-Sulfur-Optimization-Training-Coverage-Supplement-v1.0/README.md]: I'll merge to get these going over the weekend, but in the future it'd be more user-friendly to have this in the dataset description as well (i.e. re-generate the dataset).

**Comment by lilyminium** [comment]
Manually moved to error cycling.

**Pull request merged**